src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -242,6 +242,7 @@ def __init__(
                 load_sync_channel=load_sync_channel,
                 load_sync_timestamps=load_sync_timestamps,
                 experiment_names=experiment_names,
+                stream_folder=stream_folder,
```

Comment:
```suggestion
```
Hi @LiesDec 

The `self._kwargs` contains only the argument that are passed to the init. If you want to stor the `stream_folder`, you can store it in a `self._stream_folder` variable instead

Reply:
I changed it in a new commit :)